### ISSUE TITLE
I am getting the following errors in High Sierra (MacOS) when I try to do `python setup.py install`.  Could you please fix?

### ISSUE THREAD

**Issue by shyamalschandra** [jalan/pdftotext#55]
```
/usr/local/include/poppler/cpp/poppler-global.h:53:40: warning: deleted function
      definitions are a C++11 extension [-Wc++11-extensions]
    noncopyable(const noncopyable &) = delete;
                                       ^
/usr/local/include/poppler/cpp/poppler-global.h:54:57: warning: deleted function
      definitions are a C++11 extension [-Wc++11-extensions]
    const noncopyable& operator=(const noncopyable &) = delete;
                                                        ^
In file included from pdftotext.cpp:5:
/usr/local/include/poppler/cpp/poppler-page.h:39:22: warning: rvalue references
      are a C++11 extension [-Wc++11-extensions]
    text_box(text_box&&) noexcept;
                     ^
/usr/local/include/poppler/cpp/poppler-page.h:39:25: error: expected ';' at end
      of declaration list
    text_box(text_box&&) noexcept;
                        ^
                        ;
/usr/local/include/poppler/cpp/poppler-page.h:40:33: warning: rvalue references
      are a C++11 extension [-Wc++11-extensions]
    text_box& operator=(text_box&&) noexcept;
                                ^
/usr/local/include/poppler/cpp/poppler-page.h:40:36: error: expected ';' at end
      of declaration list
    text_box& operator=(text_box&&) noexcept;
                                   ^
                                   ;
4 warnings and 2 errors generated.

```

I am getting the following errors in High Sierra (MacOS) when I try to do `python setup.py install`.  Could you please fix?

**Comment by shreyas1599** [comment]
I had a similar error. Looks like it doesn't recognize C++11. Adding the CPPFLAGS fixed it for me.
Try running this:
`CPPFLAGS="-std=c++11" python setup.py install`

**Comment by shyamalschandra** [comment]
I'll check it out!  Thanks for the advice!

**Comment by jalan** [comment]
This should now be fixed in [version 2.1.4](https://pypi.org/project/pdftotext/2.1.4/)
